jekyll/_includes/snippets/pipelines-benefits.adoc

```diff
@@ -0,0 +1,10 @@
+- Use pipeline parameters to trigger [conditional workflows]({{ site.baseurl }}/2.0/pipeline-variables/#conditional-workflows).
+- Include [resuable config]({{ site.baseurl }}/2.0/reusing-config/) elements, including executors, commands, jobs, as well as packaged resuable config, known as [orbs](https://circleci.com/orbs/).
```

Comment:
Perhaps something like
Access to config version 2.1, which provides:
* more detailed config validation error messages
* re-usable config
* orbs

Reply:
Good idea